wordpress-mobile/WordPress-iOS
# [Gutenberg] File block - Implementing WP media picker source

gutenberg PR: https://github.com/WordPress/gutenberg/pull/26960
gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2805

Implementing WP Media Library source for File block

![IMG_300098FE578A-1](https://user-images.githubusercontent.com/9772967/99516882-fce6a180-298e-11eb-8ea7-559617f2fc51.jpeg)


To test:

- Create a new post.
- Add a file block.
- Press on `Choose a file`.
  - Check that the option `WordPress Media Library` is present.
- Select such option.
  - Check that all kind of files are present to be selected (such as images, videos, audios, PDFs, etc...)
- Select any of these files.
  - Check that the file is immediately added to the block.

PR submission checklist:
